cluster-autoscaler/FAQ.md

```diff
@@ -184,24 +184,23 @@ No. CPU-based (or any metric-based) cluster/node group autoscalers, like
 They are also not particularly suited to use with Kubernetes in general.
 
 
-### How Cluster Autoscaler works with Pod Priority and Preemption?
+### How does Cluster Autoscaler work with Pod Priority and Preemption?
 
-Since 1.9 CA takes pod priorities into account.
+Since version 1.1 (to be shipped with Kubernetes 1.9), CA takes pod priorities into account.
 
-Pod Priority and Preemption feature enables K8S users to schedule pods based on priorities if there is not enough resources.
-On the other hand Cluster Autoscaler makes sure that there is enough resources to run all pods.
-We introduced priority cut off to Cluster Autoscaler which tells which pods should trigger Cluster Autoscaler actions.
+Pod Priority and Preemption feature enables scheduling pods based on priorities if there is not enough resources.
+On the other hand, Cluster Autoscaler makes sure that there is enough resources to run all pods. In order to allow users to schedule "best-effort" pods, which shouldn't trigger Cluster Autoscaler actions, but only run when there are spare resources available, we introduced priority cutoff to Cluster Autoscaler.
 
-Pods with priority lower than cut off:
-* don't trigger scale ups, no new node is added in order to run them,
-* don't prevent scale downs, nodes running such pods can be deleted.
+Pods with priority lower than this cutoff:
+* don't trigger scale ups - no new node is added in order to run them,
+* don't prevent scale downs - nodes running such pods can be deleted.
```

Comment:
Above you change `scale up` to `scale-up`. I don't have any preference between the 2 forms, but this may be a good opportunity to decide on one of them and make our documentation consistent.

Reply:
According to Merriam-Webster dictionary, "scale up" is a verb and "scale-up" is a noun. There doesn't seem to be much room left for interpretation. I'll go through the doc again today and fix it in other places.